app/views/account/notification.scala

```diff
@@ -42,7 +42,11 @@ object notification {
             setting(
               correspondenceEmailNotification(),
               radios(form("correspondenceEmail"), translatedBooleanChoices)
-            )
+            ),
+            setting(
+              "Play sound when bell notification is recieved",
+              radios(form("playBellSound"), translatedBooleanChoices)
```

Comment:
this should be:       `radios(form("notification.playBellSound"), translatedBooleanChoices)`